spanner/tests/unit/test_streamed.py

```diff
@@ -99,15 +99,52 @@ def _makeListValue(values=(), value_pbs=None):
             return Value(list_value=ListValue(values=value_pbs))
         return Value(list_value=_make_list_value_pb(values))
 
+    @staticmethod
+    def _makeResultSetMetadata(fields=(), transaction_id=None):
```

Comment:
1. Please use PEP8 method names: `_make_result_set_metadata`
1. Thank your for making this a `@staticmethod`